samples/ComputerVision/readme.md

```diff
@@ -0,0 +1,48 @@
+---
+topic: sample
+languages:
+  - C# 
+products:
+  - azure
+---
+
+# Sample Solution that uses the Computer Vision SDK
+
+The projects in this code are quickstarts that show how to use Computer Vision SDK for local and remote images
+
+## Contents
+
+| File/folder | Description |
+|-------------|-------------|
+| AnalyzeImage       | Analyzes image on specified visual parameters |
+| DetectObjects | Detects objects in an image |
+| ExtractText | Looks for printed or handwritten text in an image|
+| OCR | Performs OCR on the image|
+| ComputerVision.sln | The Visual Studio solution with the above as projects|
+| `readme.md` | This README file. |
+
+
+## Prerequisites
+
+- [Visual Studio 2015](https://visualstudio.microsoft.com/) or higher
+- [An Azure Subscription Key](https://azure.microsoft.com/en-us/try/cognitive-services/?api=computer-vision) 
+
+## Setup
+
+1. Clone or download this sample repository
+2. Open the solution ComputerVision_SDK.sln in Visual Studio
+3. Download the images from [this repo](https://github.com/Azure-Samples/cognitive-services-sample-data-files/blob/master/ComputerVision/Images/) and add the Images folder as content to each project. Alternatively modify the `imageFilePath` in all projects to reflect the path of an appropriate local image.
+
+
+## Running the samples
+
+1. Right-click on the relavant project (detectObjects, extractObjects, etc) and click on 'Set as StartUp Project'
+2. In 'Program.cs' , update the line ```private const string subscriptionKey = "<your training key here>";``` with your subscription key. For example, if your subscription key is `0123456789abcdef0123456789ABCDEF`, then the line should look like ```c# private const string subscriptionKey = "0123456789abcdef0123456789ABCDEF"; ```
+3. Update the endpoint with the region you generated your endpoint for. For example, if you are using the westcentralus endpoint, you should change the line ``` computerVision.Endpoint = "https://westus.api.cognitive.microsoft.com"; ``` to  ``` computerVision.Endpoint = "https://westcentralus.api.cognitive.microsoft.com"; ```
```

Comment:
This step needs update based on the new format, now that we have a dedicated variable for this inside the Program class.